packages/SwingSet/test/vat-admin/slow-termination/slow-termination.test.js

```diff
@@ -182,7 +202,7 @@ async function doSlowTerminate(t, mode) {
   t.true(JSON.parse(kvStore.get('vats.terminated')).includes(vatID));
   // no cleanups were allowed, so nothing should be removed yet
   t.truthy(kernelStorage.kvStore.get(`${vatID}.options`));
-  t.is(remainingKV().length, 129);
+  t.is(remainingKV().length, 169);
```

Comment:
We should avoid repeating this magic number, especially since it may need occasional updating.
```suggestion
  t.is(remainingKV().length, initialKVCount);
```

Reply:
This test tells a great story, but may be overly fine-tuned. No suggestions at the moment, though.